packages/react-components/src/layout/tests/chromatic/Stack.chroma.jsx

```diff
@@ -10,57 +11,57 @@ function stories(segment) {
 stories()
     .add("default", () =>
         <Stack>
-            <div className="bg-primary-500">Alpha</div>
-            <div className="bg-primary-500">Bravo</div>
-            <div className="bg-primary-500">Charlie</div>
+            <div className="bg-primary-500"><Text>Alpha</Text></div>
```

Comment:
We shouldn't need the Text component here. I mean the devs who use our components should be able to render text without using the <Text /> component.

Reply:
Basically the Text component exist for usecases where the componens render multiple children for example :

```
                    <Button>
                        <SignoutIcon />
                        <Text>Button</Text>
                    </Button>
```

Because we use the Text component in this usecase, the Button component can internally differentiate the text and the icon and augment them accordingly.

Reply:
After discussion we might want to do this.